tests/rados/test_track_availibility_status.py

```diff
@@ -0,0 +1,604 @@
+        parts = line.split()
+        pool_name = parts[0]
+        available_data_in_dict[pool_name] = tuple(parts[1:])
+    return available_data_in_dict
+
+
+def time_to_seconds(time_str: str) -> int:
+    """
+    Method is used to convert time string (like '4d', '3h', '10m', '45s') to seconds.
+    Args:
+        time_str : Time string
+    return:
+        Converted time into seconds
+    """
+    if time_str.endswith("d"):
+        return int(time_str[:-1]) * 24 * 60 * 60
+    elif time_str.endswith("h"):
+        return int(time_str[:-1]) * 60 * 60
+    elif time_str.endswith("m"):
+        return int(time_str[:-1]) * 60
+    elif time_str.endswith("s"):
+        return int(time_str[:-1])
+    else:
+        raise ValueError("Unsupported time format. Use d, h, m, or s at the end.")
+
+
+def test_MTBF_value(client, pool_name):
+    """
+    Method is used to test the Mean Time Between Failures (MTBF) value
+    Formula - Mean Time Between Failures (MTBF) = Total uptime of pool/No of Failures
+    Args:
+        client: Client object
+        pool_name : pool name
+    Returns:
+        True -> if the expected MTBF is same as the actual MTBF
+        False -> if the expected MTBF is not same as the actual MTBF
+
+    """
+    track_available_data = get_pool_availability_status(client)
+    pool_data = track_available_data[pool_name]
+    log.info(
+        " Formula - Mean Time Between Failures (MTBF) = Total uptime of pool/No of Failures"
+    )
+    try:
+        total_up_time_pool = time_to_seconds(pool_data[0])
+        no_of_failures = int(pool_data[2])
+        expected_MTBF = time_to_seconds(pool_data[3])
+    except ValueError as e:
+        print(f"Error: {e}")
+        return 1
+    if no_of_failures == 0:
+        actual_MTBF_output = 0
+    else:
+        actual_MTBF_output = math.floor(total_up_time_pool / no_of_failures)
+    msg_output = f"Case 1: The MTBF actual output is {actual_MTBF_output}"
+    log.info(msg_output)
+
+    if actual_MTBF_output != expected_MTBF:
+        msg_err = (
+            f"Case 1: The MTBF actual output is{actual_MTBF_output} and expected is {pool_data[3]}, "
+            f"which are not same "
+        )
+        log.error(msg_err)
+        return False
+    return True
+
+
+def test_MTTR_value(client, pool_name):
+    """
+    Method is used to test the Mean Time To Recover(MTTR) value.
+    Formula - Mean Time To Recover(MTTR) = Total downtime of pool/No of Failures
+    Args:
+        client: Client object
+        pool_name : pool name
+    Returns:
+        True -> if the expected MTTR is same as the actual MTTR
+        False -> if the expected MTTR is not same as the actual MTTR
+
+    """
+    track_available_data = get_pool_availability_status(client)
+    pool_data = track_available_data[pool_name]
+    log.info(
+        " Formula - Mean Time To Recover(MTTR) = Total downtime of pool/No of Failures"
+    )
+    try:
+        total_down_time = time_to_seconds(pool_data[1])
```

Comment:
how is the total_down_time and no_of_failures on a pool calculated? 

Check how can we calculate it manually, other than relying on the `ceph osd pool availability-status` cmd. We can also validate if the results in this command o/p are correct.

Reply:
A separate tracker has been raised to analyze the details- https://issues.redhat.com/browse/RHCEPHQE-21565